client/egypt-drug-checker-dashboard/src/components/header/drop-menu/profile/Profile.component.tsx

```diff
@@ -0,0 +1,52 @@
+import TextButton from "@/components/shared/text-button/TextButton.component";
+import MUateReviewIcon from "@mui/icons-material/RateReview";
+import MUmanageAccountsIcon from "@mui/icons-material/ManageAccounts";
+import { IconButton, Typography } from "@mui/material";
```

Comment:
same comment as before

Reply:
chore(frontend): adjusting name of the typography and IconButton component to MUtypography and MUiconButton